cultivation/scripts/running/parse_run_files.py

```
@@ -29,141 +60,354 @@ def parse_fit_with_fitdecode(file_path):
+    if 'latitude' not in df.columns and 'lat' in df.columns:
+        df = df.rename(columns={'lat': 'latitude'})
+    if 'longitude' not in df.columns and 'lon' in df.columns:
+        df = df.rename(columns={'lon': 'longitude'})
+    # Ensure time is the index and is datetime-like
+    if 'time' in df.columns:
+        df = df.set_index('time')
+    if not pd.api.types.is_datetime64_any_dtype(df.index):
+        try:
+            df.index = pd.to_datetime(df.index)
+        except Exception as e:
+            if VERBOSE: print(f"[ERROR] Could not convert index to datetime: {e}")
```

Ruff (0.8.2)

Multiple statements on one line (colon)

(E701)